service/src/test/java/de/adorsys/psd2/sandbox/xs2a/testdata/TestDataFileReaderTest.java

```diff
@@ -0,0 +1,18 @@
+package de.adorsys.psd2.sandbox.xs2a.testdata;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import de.adorsys.psd2.sandbox.xs2a.testdata.domain.Transaction;
+import java.util.HashMap;
+import org.junit.Test;
+
+public class TestDataFileReaderTest {
```

Comment:
We should check at least _some_ content of the parsed objects.

Reply:
to be continued ...